Makefile

```diff
@@ -137,8 +137,8 @@ ifeq ($(PYTHON_VER),python2.7)
 	@$(SYS_PYTHON) $(VIRTUAL_BOOTSTRAP) $(VIRTUALENV_OPTS) --system-site-packages $(BLD_DIR_ENV)
 else ifeq ($(PYTHON_VER),python3.8)
 	@$(SYS_PYTHON) -m pip install --upgrade pip
-	@$(SYS_PIP) install virtualenv
```

Comment:
Curious question: Using virtualenv had worked for Apple M1 Macs for us which are also based on ARM architecture chips. Any idea why this failed for our check here (I'm pretty new to ARM arch)? Is there some difference for ubuntu CircleCI ARM nodes?

Also since we are updating the Makefile and to not have any unwanted regressions for different platform offerings. Can we try using the `venv` way like in an if check (maybe checking for arch==arm64 and os==ubuntu etc I think?) which can help us distinguish between this Linux ARM CI check and with all other ways we can use virtualenv?